waspc/src/Wasp/Util/Node/Version.hs

```diff
@@ -0,0 +1,79 @@
+module Wasp.Util.Node.Version
```

Comment:
Reminder to revert this.

Reply:
Moved this to `Wasp.Generator.Node.Version`